### ISSUE TITLE
Prevent Start Trigger "Run in parallel with the previous step" from being displayed in the first step

### ISSUE THREAD

**Issue by KennethBates** [OctopusDeploy/Issues#6723]
# Prerequisites

- [x] I have verified the problem exists in the latest version
- [x] I have searched [open](https://github.com/OctopusDeploy/Issues/issues) and [closed](https://github.com/OctopusDeploy/Issues/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed) issues to make sure it isn't already reported
- [x] I have written a descriptive issue title
- [x] I have linked the original source of this report
- [x] I have tagged the issue appropriately (area/*, kind/bug, tag/regression?)

# The bug

On the first step in a process, you can select the Start Trigger option `Run in parallel with the previous step`, which doesn't make sense in the context of it being the first step. This is available to select, and when you do so it reverts your selection back to the default `Wait for the previous step to complete, then start`, which is confusing.

## What I expected to happen

The `Run in parallel with the previous step` option wouldn't be surfaced on a step that's the first in the process.

## Steps to reproduce

1. Set the start trigger on the first step in a process to `Run in parallel with the previous step`
2. Reload the step
3. See the default setting being reverted back to

### Screen capture

![start_trigger_default_revert](https://user-images.githubusercontent.com/20388190/103725299-018c1580-5022-11eb-8b5e-c2cd89bd9b44.gif)

## Affected versions

Reproduced in **Octopus Server** 2020.5.3 and reported in 2020.5.1

## Links

Report: https://help.octopus.com/t/child-step-start-condition/26136

**Comment by droyad** [comment]
Possible duplicate of #6584

**Comment by tothegills** [comment]
Confirmed this is fixed in 2020.6.0
